docs(samples): Created initial sample for Document AI (#2019)
GoogleCloudPlatform/dotnet-docs-samples: documentai/api/README.md

# Google Cloud Document AI Sample

These samples demonstrate how to interact with the [Google Cloud Document AI API][Documentai] using C# and
the .NET client libraries to call the Cloud Document AI API.

> The samples requires [.NET Core 3.1][net-core] or later.  That means using

Reviewer: ```suggestion
The samples require [.NET 6][net-6] or later.  That means using

```

> [Visual Studio 2019](https://www.visualstudio.com/), or the command line.

Reviewer: ```suggestion
[Visual Studio 2022](https://www.visualstudio.com/), or the command line.

```

## Setup

1.  Set up a [.NET development environment](https://cloud.google.com/dotnet/docs/setup).

4.  Enable APIs for your project.
    [Click here][enable-api] to visit Cloud Platform Console and enable the Google Cloud Document AI API.

## Contributing changes

* See [CONTRIBUTING.md](../../CONTRIBUTING.md)

## Licensing

* See [LICENSE](../../LICENSE)

## Testing

* See [TESTING.md](../../TESTING.md)

[Documentai]: https://cloud.google.com/documentai/docs
[enable-api]: https://console.cloud.google.com/flows/enableapi?apiid=documentai.googleapis.com&showconfirmation=true

> [net-core]: https://www.microsoft.com/net/core

Reviewer: ```suggestion
[net-6]: https://dotnet.microsoft.com/download/dotnet/6.0

```
